api/apps/api/src/modules/scenarios/scenario.api.entity.ts

```diff
@@ -78,6 +78,9 @@ export class Scenario extends TimeUserEntityMetadata {
   @OneToOne(() => ScenarioBlm)
   scenarioBlm!: ScenarioBlm;
 
+  @Column('boolean')
+  solutionsAreLocked?: boolean;
```

Comment:
I think we should set `solutionsAreLocked` as required since it can't hold null values :thinking: 

```suggestion
  solutionsAreLocked!: boolean;
```